inference-engine/src/vpu/graph_transformer/src/middleend/passes/allocate_resources.cpp

```diff
@@ -206,10 +206,20 @@ AllocationResult runAllocator(const Model& model, EnableShapeAllocation enableSh
     //
 
     if (enableShapeAllocation == EnableShapeAllocation::YES) {
+        std::map<DimValues, int> offsets;
```

Comment:
Could this map to be a part of allocator and used when shapes are allocated? I think it'll make code a bit simpler, since you probably will get rid of offset parameter which has special value `-1`

Reply:
Made it part of allocator.